dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -239,8 +239,14 @@ private boolean problemFilter(LinkageProblem linkageProblem) {
     String targetClassName = symbol.getClassBinaryName();
     String methodName = symbol.getName();
 
-    // Skip references to Java runtime class. For example, java.lang.String.
-    if (classDumper.isSystemClass(targetClassName)) {
+    if (ClassDumper.isArrayClass(targetClassName)) {
+      return Optional.empty();
+    }
+    // Skip references from Java runtime class. For example, java.lang.String.
+    // It is possible for wrongly configured Java compiler to generate bad byte code that references
```

Comment:
I'm not sure lines 246-248 explain this code. It explains what we're not doing, not what we are. I suggest deleting them.

Reply:
Removed this if-statement and moved the explanation to array class skipping.